dotcom-rendering/src/web/browser/relativeTime/updateTimeElements.ts

```diff
@@ -7,7 +7,7 @@ export const updateTimeElement = (element: Element): void => {
 		const absoluteTime = element.getAttribute('datetime');
 		if (!absoluteTime || !relativeFormat) return;
 
-		let newTime;
+		let newTime: false | string = false;
```

Comment:
```suggestion
		let newTime: false | undefined;
```

Reply:
The `timeAgo` function returns a string or false, so we can't use undefined (at least not easily)